core/domain/topic_domain.py

```diff
@@ -29,12 +29,12 @@
 from core import utils
 from core.constants import constants
 from core.domain import change_domain
-from core.domain import fs_domain
 from core.domain import subtopic_page_domain
 
 from typing import List, Optional
 from typing_extensions import TypedDict
 
+from core.domain import fs_services # pylint: disable=invalid-import-from # isort:skip
```

Comment:
Oh, we are trying to get rid of dependencies of the domain objects on services, this goes a bit against that, can we somehow refactor this file?

Reply:
@vojtechjelinek. Maybe we can do something like this. One of the functions which uses file-system in _topic_domain.py_ is `update_thumbnail_filename`. Now instead of creating the `GcsFileSystem` instance here, it could be passed as a parameter to the function, thus allowing us to remove the call in this file. Sounds okay?

Reply:
Hmm, can you maybe move the whole function to the service file and then only set some of the things in the domain (the file will be modified outside the domain file)

Reply:
Done. I have removed one of the functions in _topic_domain.py_ which used file-system since it was not being used anywhere. Let me know if that's okay.
https://github.com/oppia/oppia/blob/2037608dc1534e41748e7ac62bedfca61213eec5/core/domain/topic_domain.py#L1175

Reply:
@vojtechjelinek One of the functions using the file-system in _topic_domain.py_ is a class method. 
https://github.com/oppia/oppia/blob/2037608dc1534e41748e7ac62bedfca61213eec5/core/domain/topic_domain.py#L1175
This can't be refactored the way we did earlier, right? (moving the functions to the service file)

Reply:
Please add a comment explaining why is it kept here.

Reply:
Done.